tests/storage/hotplug.go

```diff
@@ -1158,6 +1162,205 @@ var _ = SIGDescribe("Hotplug", func() {
 		})
 	})
 
+	Context("with limit range in namespace", func() {
```

Comment:
I think a unit test would be sufficient. WDYT?

Reply:
I think a functional test is better (yes it is slower) but we can demonstrate this actually works for various combinations of ratios in a real cluster. A unit test would just show us that indeed the ratio is 1 to 1. But we are interested in does the functionality still work when various combinations of ratios are defined in the namespace.